sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/SemanticAttributes.java

```diff
@@ -22,18 +22,227 @@
 // this is a copy of io.opentelemetry.semconv.trace.attributes.SemanticAttributes
 // because the module that contains that class is not stable,
 // so don't want to take a dependency on it
```

Comment:
I understand it's not being stable not take it as a dependency. 
If there is no new code added, i.e. it's identical, can we just use it as dependency? 
This would create a dependency on us to remember to update it regularly. It's likely to be outdated.

What kind of potential issues are you concerned about?

Reply:
> What kind of potential issues are you concerned about?

[diamond dependency problem](https://www.well-typed.com/blog/2008/04/the-dreaded-diamond-dependency-problem/), this is why azure sdk architects have been pushing back on other langs including alpha libraries in their distros